src/renderer/components/FileDisplay/FileDisplay.tsx

```diff
@@ -32,24 +36,25 @@ const FileDisplay = ({ entry, treeLevel }: Props) => {
           paddingLeft: `${treeLevel! * 2}rem`,
         }}
       >
-        {entry.directory.isDirectory ? (
-          <FolderIcon color={statusColor} />
+        {item.isDirectory ? (
+          <FolderIcon color={color} />
         ) : (
-          <FileIcon color={statusColor} />
+          <FileIcon color={color} />
         )}
-        <Typography ml='0.75rem'>{entry.name}</Typography>
+        <Typography ml='0.75rem'>{path.basename(item.filepath)}</Typography>
       </Styled.DataField>
       <Styled.DataField sx={{ width: widths[1] }}>
         <Typography variant='body2'>
-          {getDateString(
-            entry.details.dateModifiedMs,
-            i18n.language as SupportedLangCode
-          )}
+          {item.stats &&
+            getDateString(
+              item.stats.mtimeSeconds * 1000,
+              i18n.language as SupportedLangCode
+            )}
         </Typography>
       </Styled.DataField>
```

Comment:
@LostInStatic The `DataField` needs to be in the tree even if the `item.stats` are `undefined` so that the layout is preserved?

Reply:
Yes. It happens when the file is deleted from the workdir. We could display a message ("file removed" or something like that) if the reason the field is empty might be unclear for the user.

Reply:
Should be fine, can always add it later. 👍